### PR TITLE
Naming alignment

### PR DESCRIPTION
**Pull request by ValentinaKats**
Align mentioned names with official naming

### CODE REVIEW

File: documentation/library_guide/conf.py [hunk 1]
```diff
@@ -15,7 +15,8 @@
 #===============================================================================
 
 ##  Content:
-##     Intel(R) DAAL configuration file for the Sphinx documentation builder
+## Intel(R) Data Analytics Acceleration Library (Intel(R) DAAL) configuration
```
Comment (AlexVeprev) [inline documentation/library_guide/conf.py:18]: Seems this comment is not actual at all, why should `Intel(R) DAAL` be mentioned in this file?

Comment (ValentinaKats) [reply documentation/library_guide/conf.py:18]: Thanks for noticing! I'll clarify.

**Review by timmiesmith — COMMENTED** [review]
@ValentinaKats, other than the issue @AlexVeprev noted the changes look good to me.

File: documentation/library_guide/conf.py (hunk 1 above)
Comment (ValentinaKats) [reply documentation/library_guide/conf.py:18]: Looks like Intel(R) DAAL is mentioned here on purpose. Some lines in the script are related to DAAL. Let's keep it.

Comment (AlexVeprev) [reply documentation/library_guide/conf.py:18]: Please check it separately more carefully.

**Review by AlexVeprev — APPROVED** [review]

**Pull request merged**